plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -625,6 +639,24 @@ private function count_orders_by_status( $status ): int {
 	 * @return boolean TRUE when the blank state should be rendered, FALSE otherwise.
 	 */
 	private function should_render_blank_state(): bool {
+		/**
+		 * Filters if we should render blank state, allowing for custom count queries to be used
+		 *
+		 * @since x.x.x
+		 *
+		 * @param null           $should_render_blank_state null will use the build in counts, and sending a boolean will overwrite it
+		 * @param object         ListTable The current instance of the class.
+		*/
+		$should_render_blank_state = apply_filters(
+			'woocommerce_' . $this->order_type . '_list_table_render_blank_state',
```

Comment:
Same, slight suggestion in naming filter:
```suggestion
			'woocommerce_' . $this->order_type . '_list_table_should_render_blank_state',
```